src/Material.cpp

```diff
@@ -1313,7 +1452,7 @@ void Material::checkSigmaT() {
  *          scattering cross-sections and chi for all energy groups.
  * @return character array of this Material's attributes
  */
-std::string Material::toString() {
+std::string MacroMaterial::toString() {
```

Comment:
This seems reasonable, but I'd prefer to leave it for a future PR, unless you think it's really important.